public/js/plugins/github/github-modal.controller.js

```diff
@@ -38,6 +38,10 @@ module.exports =
   function setRepos() {
     vm.title = 'Repositories';
     vm.step  = 2;
+    vm.pagination = githubService.config.pagination;
+    if (!vm.totalItems) {
+      vm.totalItems = vm.pagination.last.page * vm.itemsPerPage;
```

Comment:
We'll set the `totalItems` variable in the initial request and not in any following requests since when a user has paginated to the last page there is no `last` property and the app will crash.